libs/SmartStore/SmartStoreTests/SFSmartStoreFullTextSearchTests.h

```diff
@@ -27,5 +27,12 @@
 
 #import "SFSmartStoreTestCase.h"
 
-@interface SFSmartStoreFullTextSearchTests : SFSmartStoreTestCase 
+@interface SFSmartStoreFullTextSearchTests : SFSmartStoreTestCase
+
+#define kTestStore            @"testSmartStoreFullTextSearchStore"
```

Comment:
Just for my understanding, what's the difference between  #define ans static const NSString * ?